components/scream/src/physics/p3/p3_functions_droplet_self_coll_impl.hpp

```diff
@@ -0,0 +1,35 @@
+#ifndef P3_FUNCTIONS_DROPLET_SELF_COLL_IMPL_HPP
+#define P3_FUNCTIONS_DROPLET_SELF_COLL_IMPL_HPP
+
+#include "p3_functions.hpp" // for ETI only but harmless for GPU
+
+namespace scream {
+namespace p3 {
+
+/*
+ * Implementation of p3 droplet self collection function. Clients should NOT
+ * #include this file, but include p3_functions.hpp instead.
+ */
+
+template <typename S, typename D>
+KOKKOS_FUNCTION
+void Functions<S,D>
+::droplet_self_collection(const Spack& rho, const Spack& inv_rho,
```

Comment:
It looks like most of these arguments aren't even used? You should remove the ones that aren't used if we are going to assume iparam=3.

Reply:
Is this just a matter of practice, or is it for silencing "unused var" warnings? I could see that making things more difficult for someone who adds another setting to this model (unless we're not going to have them revisit this same function). Some folks use UNUSED() macros to indicate that arguments aren't used, for example.

Reply:
You might be able to use anonymous argument to remove the warnings.